### ISSUE TITLE
Couldn't see preview for todo

### ISSUE THREAD

**Issue by neelabalan** [fabiospampinato/notable#151]
I couldn't see the preview for the todo in markdown in the application. Tried installing the latest release in both linux as well as windows.

**Comment by pylight** [comment]
Same here (installed on Ubuntu 18.04 with deb package). State of the todos is not visible in preview.
Is it a missing feature or a bug?

![screenshot](https://i.imgur.com/VSuojOM.png)

Thanks for the great app btw ;)

**Comment by johnletey** [comment]
Duplicate of #19.

**Comment by neelabalan** [comment]
> Same here (installed on Ubuntu 18.04 with deb package). State of the todos is not visible in preview.
> Is it a missing feature or a bug?
> 
> ![screenshot](https://camo.githubusercontent.com/13a7179541e97bf659480dbe9caf6f873b1e0e72/68747470733a2f2f692e696d6775722e636f6d2f5653756f6a4f4d2e706e67)
> 
> Thanks for the great app btw ;)

Its a bug. You can see other applications like typora display the checkbox in preview. Yeah, This is really a great markdown note taking app.

**Comment by neelabalan** [comment]
> Same here (installed on Ubuntu 18.04 with deb package). State of the todos is not visible in preview.
> Is it a missing feature or a bug?
> 
> ![screenshot](https://camo.githubusercontent.com/13a7179541e97bf659480dbe9caf6f873b1e0e72/68747470733a2f2f692e696d6775722e636f6d2f5653756f6a4f4d2e706e67)
> 
> Thanks for the great app btw ;)

yeah notable is a great app. Appreciate including the checkbox feature in the next release. @fabiospampinato
